### DISCUSSION TITLE
ESP32-2432S028R 2.8" - how to get into configuration mode (edit wallet or wifi)

### DISCUSSION THREAD

**Discussion by wenshenwang0416** [BitMaker-hub/NerdMiner_v2#187] [Q&A]
Hi all, I am using this 2.8" miner. How to get into configurati9n mode? 
I tried both buttons, held one and power up, but the screen turned black.
Please help.

**Comment by nitroxgas** [comment]
Try to long-press the button that changes the screens on startup. This should reset the configurations and force the wifi to access point mode again.

**Reply by wenshenwang0416** [reply]
Thanks!! Got it now.
One more question, how to turn off the screen? I short pressed the other button but it automatically restarted. Double pressed also did'nt work.

**Reply by canadamadman** [reply]
i have the same similar issue with esp32 wroom bord. when i do the reset i can only seem to change wifi settings and not my btc or pool
